packages/ui-breadcrumb/src/Breadcrumb/BreadcrumbLink/index.tsx

```diff
@@ -60,8 +60,6 @@ class BreadcrumbLink extends Component<BreadcrumbLinkProps> {
 
     return (
       <Link
-        // @ts-expect-error ts-migrate(2339) FIXME: Property 'element' does not exist on type 'Breadcr... Remove this comment to see the full error message
```

Comment:
On line 40, add `@tsProps` to the annotation. We have to add it to each component we fully typed so the doc generation knows that it can render the props from the TS types safely instead of the old PropTypes :)

/**
---
parent: Breadcrumb
id: Breadcrumb.Link
---
@tsProps
**/